test/mpi-cpp-fortran/base/mpi-cpp-fortran.cmake

```diff
@@ -0,0 +1 @@
+include("${CMAKE_CURRENT_LIST_DIR}/mpi-cpp-fortran-targets.cmake")
```

Comment:
What is this file for?

Reply:
these are the targets exported by the base project to the downstream project.  Typically this would include all TPL targets but doesn't here

Reply:
If you are using:

```
install(TARGETS mpi-cpp-fortran
  EXPORT mpi-cpp-fortran-targets)
```

doesn't that create `mpi-cpp-fortran-targets.cmake` where the cmake logic is installed?

Does it need to exist in the source dir?

Reply:
Sorry, I was confused, deleted a question about mpi-cpp-fortran-targets.cmake

Reply:
Sorry I wasnt clear enough. I understand what the `mpi-cpp-fortran-targets.cmake` is for. Its generated by CMake for the project created targets. I was talking about this whole file, `test/mpi-cpp-fortran/base/mpi-cpp-fortran.cmake`. It is not needed and also not installed. I ran the test script before and after deleting this file and the test projects still pass.

My guess is it was a stepping stone to getting the real file working, `test/mpi-cpp-fortran/base/mpi-cpp-fortran-config.cmake`.